src/store/store.tsx

```diff
@@ -0,0 +1,31 @@
+// src/app/store.js
+import { configureStore } from '@reduxjs/toolkit';
+import expenseFormReducer from '../modules/expense/expenseFormSlice';
+import dayjs, { Dayjs } from 'dayjs';
+
+interface RootState {
+  expenseForm: {
+    category: string[];
+    textValue: string;
+    selectedWallet: string;
+    showAttachmentComponent: boolean;
+    showContinueButton: boolean;
+    switchChecked: boolean;
+    toggleContinueButton: boolean;
+    selectedFrequency: string;
```

Comment:
Change to 'paymentFrequency', the type should be enum with supported frequencies